tox.ini

```diff
@@ -21,6 +21,9 @@ commands =
 [testenv:build_docs]
 basepython = python3.7
 extras = all,docs
+deps =
+    astroscrappy
+    zeep
```

Comment:
Looks like we need to add   beautifulsoup4 as well

Reply:
Yaa.